index.html

```diff
@@ -38,244 +37,137 @@
+          <p><a class="btn btn-primary btn-lg" href="about.html" role="button">Learn more &raquo;</a></p>
+        </div>
+      </div>
 
-          <div class="codesample-left">
-            <span class="code-title">On the original machine</span>
-            <div class="highlight">
-              <pre><code class="console">$ pip install reprozip
-$ reprozip <span class="s">trace</span> <span class="na">./myexperiment -my --options inputs/somefile.csv other_file_here.bin</span>
-<span class="nt">experiment: 0%... 25%... 50%... 75%... 100%
-result: 42.137</span>
-<span class="c">Configuration file written in .reprozip/config.yml
-Edit that file then run the packer -- use 'reprozip pack -h' for help</span>
-$ reprozip <span class="s">pack</span> my_experiment.rpz
-<span class="c">[REPROZIP] 17:26:42.588 INFO: Creating pack my_experiment.rpz...
-[REPROZIP] 17:26:42.589 INFO: Adding files from package coreutils...
-[REPROZIP] 17:26:42.601 INFO: Adding files from package libc6...
-[REPROZIP] 17:26:42.906 INFO: Adding other files...
-[REPROZIP] 17:26:43.450 INFO: Adding metadata...</span></code></pre>
+      <div class="container">
+        <div class="card-deck">
+          <div class="card">
+            <div class="card-body">
+              <h3 class="card-title">No lock-in</h3>
+              <p class="card-text">ReproZip works locally on your computer, server, or high performance computing clusters. Once you make an <code>.rpz</code> bundle, you can reproduce it with a variety of unpacker plug-ins (e.g. Docker, Vagrant, Singularity) or in the cloud with <a href="https://github.com/vida-nyu/reproserver">ReproServer</a>.</p>
             </div>
-          </div>
-
-          <div class="codesample-bottom">
-            <div class="codesample-arrow"></div>
-            <div class="codesample-right">
-              <span class="code-title">On another machine</span>
-              <div class="highlight">
-                <pre><code class="console">$ pip install reprounzip[all]
-$ reprounzip vagrant <span class="s">setup</span> my_experiment.rpz mydirectory
-<span class="c">Bringing machine 'default' up with 'virtualbox' provider...
-==> default: Importing base box 'remram/debian-7-amd64'...
-==> default: Booting VM...
-==> default: Machine booted and ready!
-==> default: Running provisioner: shell...</span>
-$ reprounzip vagrant <span class="s">run</span> mydirectory
-<span class="nt">experiment: 0%... 25%... 50%... 75%... 100%
-result: 42.137</span>
-$ reprounzip vagrant <span class="s">upload</span> /tmp/new_config:global-config
-$ reprounzip vagrant <span class="s">run</span> mydirectory --cmdline <span class="na">./myexperiment --other --options inputs/somefile.csv</span>
-<span class="nt">experiment: 0%... 25%... 50%... 75%... 100%
-result: -17.814</span></code></pre>
-              </div>
+            <div class="card-footer">
+              <a class="btn btn-outline-primary" href="#" role="button"> Try out ReproZip today &raquo;</a>
```

Comment:
This link doesn't go anywhere

Reply:
Good catch! I will fix this to go to the right place.